x-pack/plugins/security_solution/server/lib/entity_analytics/risk_engine/risk_engine_data_client.test.ts

```diff
@@ -60,343 +60,347 @@ jest.spyOn(transforms, 'createTransform').mockResolvedValue(Promise.resolve());
 jest.spyOn(transforms, 'scheduleTransformNow').mockResolvedValue(Promise.resolve());
 
 describe('RiskEngineDataClient', () => {
-  for (const useDataStreamForAlerts of [false, true]) {
-    const label = useDataStreamForAlerts ? 'data streams' : 'aliases';
```

Comment:
Sorry for the big noise, in my iterations, I changed the `for` for `describe.each`, removing whitespace changes in the PR might make it easier to review.